server/sonar-main/src/main/java/org/sonar/application/cluster/ClusterAppStateImpl.java

```diff
@@ -27,20 +27,23 @@
 import javax.annotation.Nonnull;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
-import org.sonar.application.AppState;
 import org.sonar.application.AppStateListener;
 import org.sonar.application.config.AppSettings;
+import org.sonar.cluster.localclient.HazelcastClient;
 import org.sonar.process.ProcessId;
-import org.sonar.process.ProcessProperties;
 
-public class AppStateClusterImpl implements AppState {
-  private static Logger LOGGER = LoggerFactory.getLogger(AppStateClusterImpl.class);
+import static org.sonar.cluster.ClusterProperties.CLUSTER_ENABLED;
+import static org.sonar.cluster.ClusterProperties.CLUSTER_LOCALENDPOINT;
+import static org.sonar.cluster.ClusterProperties.CLUSTER_MEMBERUUID;
+
+public class ClusterAppStateImpl implements ClusterAppState {
+  private static Logger LOGGER = LoggerFactory.getLogger(ClusterAppStateImpl.class);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Make the "LOGGER" logger private static final. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1312)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this field "LOGGER" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS3008)